pkgs/development/python-modules/csv2md/default.nix

```
@@ -0,0 +1,24 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+}:
+
+buildPythonPackage rec {
+  pname = "csv2md";
+  version = "1.1.1";
+
+  src = fetchFromGitHub {
+    owner = "lzakharov";
+    repo = pname;
```

```suggestion
    repo = "csv2md";
```
pname and repo are semantically different, so please avoid reusing pname.